braindecode/models/tidnet.py

```diff
@@ -0,0 +1,266 @@
+from torch import nn
+from math import ceil
+from torch.nn import init
+from torch.nn.utils import weight_norm
+from .modules import Ensure4d, Expression
+
+import torch
+
+
+class Expand(nn.Module):
+    def __init__(self, axis=-1):
+        super().__init__()
+        self.axis = axis
+
+    def forward(self, x):
+        return x.unsqueeze(self.axis)
+
+
+class Flatten(nn.Module):
```

Comment:
there is nn.Flatten from PyTorch, do you know why this is needed? try using nn.Flatten instead (with start_dim=1)